src/benchmarks/gc/README.md

```diff
@@ -429,7 +463,7 @@ This is the build output of coreclr that is used to run benchmarks.
 
 These are specified in the `coreclrs` section of a benchfile.
 
-This can be found in a directory like `bin/tests/Windows_NT.x64.Release/Tests/Core_Root` (adjust for different OS or architecture) of a coreclr repository. (The Core_Root can be moved anywhere and doesn't need to remain inside the coreclr repository.)
+This can be found in a directory like `bin/tests/Windows_NT.x64.Release/Tests/Core_Root` (adjust for different OS or architecture) of a coreclr repository. (The Core_Root can be moved anywhere and doesn't need to remain inside the coreclr repository).
```

Comment:
Does this seem to imply Windows only?

Reply:
Windows is just an example in this case. You need a Core_Root for Linux testing as well.

Reply:
I think we could add two examples, one for Windows and one for Linux?